src/main/scala/org/ergoplatform/nodeView/wallet/persistence/OffChainRegistry.scala

```diff
@@ -0,0 +1,67 @@
+package org.ergoplatform.nodeView.wallet.persistence
+
+import org.ergoplatform.nodeView.history.ErgoHistory
+import org.ergoplatform.wallet.boxes.TrackedBox
+
+/**
+  * Holds version-agnostic indexes (such as off-chain boxes) in runtime memory.
```

Comment:
I see it also holds `onChainBalances` and `height` that are not version-agnostic (and it is not clear what do you mean by `version`)

Reply:
In this case `onChainBalances` appear to be version-agnostic as it can be updated in result of an off-chain spending.